client/generated_longhorn_condition.go

```diff
@@ -1,87 +0,0 @@
-package client
-
-const (
-	LONGHORN_CONDITION_TYPE = "longhornCondition"
-)
-
-type LonghornCondition struct {
```

Comment:
Just double checking, we don't need this file anymore?

Reply:
This code is auto-generated by https://github.com/longhorn/longhorn-manager/blob/master/client/README.md
After following the above step, this file is gone. I am not sure if it is a problem.
How do you think?  @shuo-wu

Reply:
It shouldn't be a problem then.